### ISSUE TITLE
Update ACSF connector module to v2.61

### ISSUE THREAD

**Issue by blairlearn** [NCIOCPL/cgov-digital-platform#2450]
1. pull down the latest prod tag from ncigov artifact repo
2. Create a test branch
3. update the module composer update drupal/acsf --with-all-dependencies
4. Diff the files in modules/contrib/acsf/acsf_init/lib and the same named folder in our repo under /acsf_convert/files. (Basically cloud and factory hooks, stuff from docroot/sites)
5. Update any files.
6. Run the acsf-verify command from the ACE->ACSF conversion steps. I am too lazy to look it up.
7. Then make a branch against our NCIOCPL repo, run the update, then update the acsf-convert files.
8. Throw away the test branch from ncigov
----
Basically acsf-verify checks their hooks and some of the sites files to ensure it matches the lib folder.
If their stuff does not match what we have it errors out. Heck the actual deploy to ACSF errors out as well. (edited) 
There are some other things they can touch, but it looks like they usually dont. It really all comes down to looking at the code in the acsf-init Drupal command and seeing what they would change
